script/Ciel/Day 6/06_01_CIEL06_2.txt

```diff
@@ -41,7 +41,7 @@ After all, I am partially to blame for the fact that you did not contact us."
 [sha:3ebf85f91ae52521d671a32e40643205a1067182]{
 -- Page 12008, Offset 13423. Glued. Mods: @x.
 -- あれがあれば、ずぼらな兄さんでも連絡ぐらいはしたのではありませんか？」
- If you'd had that, surely even a negligent brother such as yourself would have at least tried to contact me, don't you think?"
+ If you had that, surely even a negligent brother such as yourself would have at least tried to contact me, don't you think?"
```

Comment:
had² correct again p sure

Reply:
oh these are all probably dupes?

Reply:
Ah no there's a bunch of 
>'d had
>'s is
>'re are
>>
All of which I thought were typos so I edited all of them out.
![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/95a124d9-4250-4957-a6a4-d7749a72dbec)